main/go.mod

```diff
@@ -0,0 +1,21 @@
+module github.hpe.com/pablo-gon-sanchez/go-github-app-boilerplate/main
```

Comment:
is there a way we can genericize this module name so it's not specific to local directory structure? If not, then I think for the purposes of this repo the module should be
```
module github.com/sharkysharks/go-github-app-boilerplate
```
you don't need to have `/main` on the end, the modules will work the same inside the main directory.

Reply:
I changed it to not use the main, only thing is the executable file has changed its name, check if you like that please?